### PR TITLE
Minor learning2learn fixes

### PR DESCRIPTION
**Pull request by albi3ro**
I changed a few things to improve the reproducibility of output.

Just a few variable names, some spelling, and printed values.

page render is [here](https://3647-214003948-gh.circle-artifacts.com/0/_build/html/demos/learning2learn.html)

### CODE REVIEW

File: demonstrations/learning2learn.py [hunk 1]
```diff
@@ -919,7 +920,6 @@ def train_step(graph_cost):
 lstm_losses = [new_cost(tf.reshape(guess, shape=(2, n_layers))) for guess in guesses]
 
 fig, ax = plt.subplots()
```
Comment (albi3ro) [inline demonstrations/learning2learn.py:922]:
```suggestion
fig, ax = plt.subplots()

```

File: demonstrations/learning2learn.py [hunk 2]
```diff
@@ -674,7 +674,7 @@ def train_step(graph_cost):
 plt.legend()
 plt.ylabel("Cost function", fontsize=12)
 plt.xlabel("Iteration", fontsize=12)
-ax.set_xticks([0, 5, 10, 15, 20]);
+ax.set_xticks([0, 5, 10, 15, 20])
```
Comment (albi3ro) [inline demonstrations/learning2learn.py:677]:
```suggestion
ax.set_xticks([0, 5, 10, 15, 20]);
```

File: demonstrations/learning2learn.py [hunk 3]
```diff
@@ -823,6 +823,7 @@ def train_step(graph_cost):
 
 # Shuffle the dataset
 import random
+random.seed(1234)
```
Comment (glassnotes) [inline demonstrations/learning2learn.py:826]: Thank you!!

File: demonstrations/learning2learn.py [hunk 4]
```diff
@@ -1061,6 +1062,53 @@ def call(self, inputs):
 model.summary()
 
 
+##############################################################################
```
Comment (glassnotes) [inline demonstrations/learning2learn.py:1065]: Good catch!

**Review by glassnotes — APPROVED** [review]
Thanks @albi3ro ! Looks great :tada: 

(Please modify the "Last updated" date at the top of the page before merging :slightly_smiling_face: )

**Comment by stfnmangini** [comment]
> @stfnmangini want to double-check these fixes?

Hi @albi3ro! Thanks for the fixes :)
They look great!

**Pull request merged**